test/Microsoft.AspNet.Razor.Runtime.Test/TagHelpers/TagHelperExecutionContextTest.cs

```diff
@@ -257,7 +259,7 @@ public void AddMinimizedHtmlAttribute_MaintainsHTMLAttributes()
         public void AddMinimizedHtmlAttribute_MaintainsHTMLAttributes_SomeMinimized()
         {
             // Arrange
-            var executionContext = new TagHelperExecutionContext("input", selfClosing: true);
+            var executionContext = new TagHelperExecutionContext("input", tagMode: TagMode.SelfClosing);
```

Comment:
Should use `StartTagOnly` in at least one test of this type per test class. That is, don't always change `selfClosing: true` to `tagMode: TagMode.SelfClosing` or the shorter `TagMode.SelfClosing` in the updated test classes.